### PR TITLE
mobile top nav updates

### LINKED ISSUE #4264: Mobile side nav
Figma: 
https://www.figma.com/file/RMXc7jUWiC6amxz9Q2JK5L/Onboarding-and-Wallet-UX?node-id=1037%3A859

### PR DESCRIPTION
**Pull request by andrewdamelio**
closes #4264

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 💪

 This link will always point to this PR's latest changes: https://peek.run/vv2h2g7

File: packages/augur-ui/src/modules/app/components/app.tsx [hunk 1]
```diff
@@ -99,16 +100,18 @@ export default class AppView extends Component<AppProps> {
...
       title: 'Reporting',
       route: REPORTING,
+      requireLogin: false,
     },
     {
       title: 'Create',
       route: CREATE_MARKET,
       requireLogin: true,
-      disabled: this.props.universe.forkingInfo,
+      disabled: this.props.universe.forkingInfo !== null,
```
Comment (bthaile) [inline packages/augur-ui/src/modules/app/components/app.tsx]: this needs to be !!this.props.universe.forkingInfo

it's possible to be undefined

File: packages/augur-ui/src/modules/app/components/app.tsx [hunk 2]
```diff
@@ -178,7 +181,7 @@ export default class AppView extends Component<AppProps> {
       updateMobileMenuState(MOBILE_MENU_STATES.CLOSED);
     }
     if (universe.forkingInfo !== nextProps.universe.forkingInfo) {
-      this.sideNavMenuData[1].disabled = nextProps.universe.forkingInfo;
+      this.sideNavMenuData[5].disabled = nextProps.universe.forkingInfo !== null;
```
Comment (bthaile) [inline packages/augur-ui/src/modules/app/components/app.tsx]: same here `!!nextProps.universe.forkingInfo`

**Review by bthaile — CHANGES_REQUESTED** [review]
need to fix the check on forkingInfo. code looks otherwise

**Review by bthaile — APPROVED** [review]
LGtm

**Pull request merged**